src/main.rs

```diff
@@ -76,10 +82,164 @@ fn color_to_index(color: &[u8]) -> u8 {
     }).min_by_key(|&(_, diff)| diff).expect("4 components").0 as u8
 }
 
+fn get_target_json(url: &str) -> Result<TargetJson, Box<Error>> {
+    let new_target: serde_json::Value = reqwest::get(url)?.json()?;
+    let new_target = new_target.as_object().ok_or("Json format error")?;
+    macro_rules! tr {
+        ($e:expr) => {
+            ($e).ok_or("Json format error")?
+        }
+    }
+
+    let fallbacks = || -> Result<Vec<String>, Box<Error>> {
+        Ok(tr!(tr!(tr!(new_target.get("fallbacks")).as_array())
+            .iter()
+            .map(|x| x.as_str().map(|x| x.to_string()))
+            .collect::<Option<Vec<String>>>()))
+    };
+    let fallbacks = fallbacks().unwrap_or_default();
```

Comment:
The fallbacks field is optional. If it exists, it expects a array of url strings to other json configs.